specs/UrbanMobility/doc/introduction.md

```diff
@@ -17,3 +17,23 @@
  Entity Attributes (Properties or Relationships) are subject to the restrictions defined by the
  [GTFS specification](https://developers.google.com/transit/gtfs/reference/#term-definitions)
  If an Attribute is an enumeration its value shall be provided as per the GTFS specification (not LinkedGTFS). 
+
+## Summary of Entity mappings with GTFS
+
+
+| GTFS Feed Member Name                                                                           | NGSI Entity Type                                                                                                        |
+|:----------------------------------------------------------------------------------------------- |:----------------------------------------------------------------------------------------------------------------------- |
+| [agency.txt](https://developers.google.com/transit/gtfs/reference/#agencytxt)                   |   [gtfs:Agency](./Agency/doc/spec.md)                                                                                   |
```

Comment:
Should these pathes start with "../"(./Agency/doc and so on)?

Reply:
well spotted, yes!